packages/react/src/components/Alert/Alert.tsx

```diff
@@ -108,6 +130,7 @@ class Alert extends UIComponent<ReactProps<AlertProps>> {
     return (
       <ElementType
         className={classes.root}
+        onFocus={this.handleFocus}
```

Comment:
Shouldn't this be added on the action, instead of the root element?

Reply:
The Alert itself as far as I understand is not focusable

Reply:
Current points:
- as I see the whole `Alert` is not focusable by [spec](https://www.w3.org/TR/wai-aria-practices/examples/alert/alert.html), so currently only `action` slot can be focused
- `focus` event will be propogated from the `action` slot

If we will have issues with this in future, i.e. the whole `Alert` can be focused we can separate values to `isFromKeyboard` & `isAlertFromKeyboard`.

Reply:
Make sense, I understand that it will be propagated, but my point was more in a sense that, if the content of the alert has focusable event, it would in the same way behave. But we can anyway use the `':focus'` selector together with the `isFromKeyboard` state value, so it should be ok.